From 1c9a8fbaf70f0647e4913dcaa864ba41132fc363 Mon Sep 17 00:00:00 2001
From: Yeeun Baek <bye0520@naver.com>
Date: Sat, 30 Jun 2018 02:06:44 +0900
Subject: [PATCH] ConceptSelection: Change location

Signed-off-by: Yeeun Baek <bye0520@naver.com>
---
 kk/src/main/ConceptSelection.java | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/kk/src/main/ConceptSelection.java b/kk/src/main/ConceptSelection.java
index 87eeb82..26a9b67 100644
--- a/kk/src/main/ConceptSelection.java
+++ b/kk/src/main/ConceptSelection.java
@@ -34,7 +34,7 @@ public class ConceptSelection extends JFrame implements ActionListener {
 
 		// 전체 버튼
 		btnAll.setIcon(new ImageIcon(SeasonSelection.class.getResource("/Photos/All.jpg")));
-		btnAll.setBounds(12, 137, 158, 140);
+		btnAll.setBounds(12, 107, 158, 140);
 		conceptPane.add(btnAll);
 		// 버튼 투명화
 		btnAll.setBorderPainted(false);
@@ -42,35 +42,35 @@ public class ConceptSelection extends JFrame implements ActionListener {
 
 		// 데이트 버튼
 		btnDate.setIcon(new ImageIcon(SeasonSelection.class.getResource("/Photos/Date.jpg")));
-		btnDate.setBounds(182, 137, 140, 140);
+		btnDate.setBounds(182, 107, 140, 140);
 		conceptPane.add(btnDate);
 		btnDate.setBorderPainted(false);
 		btnDate.setContentAreaFilled(false);
 
 		// 홍대 버튼
 		btnHongik.setIcon(new ImageIcon(SeasonSelection.class.getResource("/Photos/Hong.jpg")));
-		btnHongik.setBounds(22, 287, 140, 140);
+		btnHongik.setBounds(22, 257, 140, 140);
 		conceptPane.add(btnHongik);
 		btnHongik.setBorderPainted(false);
 		btnHongik.setContentAreaFilled(false);
 
 		// 강남 버튼
 		btnGangnam.setIcon(new ImageIcon(SeasonSelection.class.getResource("/Photos/Gang.jpg")));
-		btnGangnam.setBounds(182, 287, 140, 140);
+		btnGangnam.setBounds(182, 257, 140, 140);
 		conceptPane.add(btnGangnam);
 		btnGangnam.setBorderPainted(false);
 		btnGangnam.setContentAreaFilled(false);
 
 		// 해커톤 버튼
 		btnHack.setIcon(new ImageIcon(SeasonSelection.class.getResource("/Photos/Hack.jpg")));
-		btnHack.setBounds(22, 437, 140, 140);
+		btnHack.setBounds(22, 407, 140, 140);
 		conceptPane.add(btnHack);
 		btnHack.setBorderPainted(false);
 		btnHack.setContentAreaFilled(false);
 
 		// 면접 버튼
 		btnInterv.setIcon(new ImageIcon(SeasonSelection.class.getResource("/Photos/Inter.jpg")));
-		btnInterv.setBounds(182, 437, 140, 140);
+		btnInterv.setBounds(182, 407, 140, 140);
 		conceptPane.add(btnInterv);
 		btnInterv.setBorderPainted(false);
 		btnInterv.setContentAreaFilled(false);
@@ -78,14 +78,14 @@ public class ConceptSelection extends JFrame implements ActionListener {
 		//안내문구
 		label.setFont(new Font("Rix장미의유혹Basic Regular", Font.ITALIC, 25));
 		label.setHorizontalAlignment(SwingConstants.CENTER);
-		label.setBounds(30, 58, 302, 32);
+		label.setBounds(30, 41, 302, 32);
 		conceptPane.add(label);
 
 		conceptFrame.getContentPane().add(conceptPane);
 		conceptFrame.setVisible(true);
 
 		backBtn.setIcon(new ImageIcon(SeasonSelection.class.getResource("/Photos/Back.png")));
-		backBtn.setBounds(30, 50, 20, 40);
+		backBtn.setBounds(12, 29, 30, 50);
 		conceptPane.add(backBtn);
 		backBtn.setBorderPainted(false);
 		backBtn.setContentAreaFilled(false);
-- 
2.16.2.windows.1

